API.yaml

```diff
@@ -489,6 +489,86 @@ paths:
             application/json:
               schema:
                 type: string
+  /experiments/{expUUID}/outputs/annotations/{outputID}:
+    get:
+     summary: Get the collection of annotations categories for an experiment.
+     operationId: getExperimentAnnotationCategories
+     tags:
+      - Annotations
+     parameters:
+      - name: expUUID
+        in: path
+        description: The UUID of the experiment in the server
+        required: true
+        schema:
+          type : integer
+          format: int128
+      - name: outputID
+        in: path
+        description: ID of the output provider to query
+        required: true
+        schema:
+          type: string
+     responses:
+        200:
+          description: Returns a list of annotation categories for this trace
+          content:
+            application/json:
+              schema:
+                type: array
+                items:
+                  $ref: '#/components/schemas/AnnotationCategory'
+        404:
+          description: No such experiment
+          content:
+            application/json:
+              schema:
+                type: string
+    post:
+     summary: Get a list of specific annotations from this experiment
+     operationId: getExperimentAnnotations
+     tags:
+      - Annotations
+     parameters:
+      - name: expUUID
+        in: path
+        description: The UUID of the experiment in the server
+        required: true
+        schema:
+          type : integer
+          format: int128
+      - name: outputID
+        in: path
+        description: ID of the output provider to query
+        required: true
+        schema:
+          type: string
+     requestBody:
+        description: Query parameters to fetch the annotations.
+          Usually contains these parameters, but not limited to
+          - list of outputId entries
+          - list of categoryID
+          - startTime
```

Comment:
This I'm not sure, I think we should send timeRequested as an array instead of start, end and resolution